Cargo.toml

```diff
@@ -54,6 +54,7 @@ predicates = ">=2.0, <4.0"
 pyo3 = "0.21.2"
 quick-xml = "0.32"
 rand = "0.8"
+rayon = "1.10"
 rayon-core = "1.11"
```

Comment:
I think you need to update the version to 1.12.1 to make MSV CI happy

Reply:
Fixed!